zerver/views/home.py

```diff
@@ -190,6 +179,10 @@ def home_real(request):
 
     request.session[translation.LANGUAGE_SESSION_KEY] = default_language
 
+    # Set status dict with last time visit for user
+    status_dict = UserPresence.get_status_dict_by_user(user_profile)
```

Comment:
This does a more expensive query than needed, since it'll also pull information from the UserProfile table.
I would do 
`UserPresense.objects.filter(user_profile=user).values_list('timestamp', flat=True).first()`.
And since it is just one line we can just put it inline in the page_params dict below